elasticsearch/values.yaml

```diff
@@ -118,6 +118,9 @@ readinessProbe:
   successThreshold: 3
   timeoutSeconds: 5
 
+# https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-health.html#request-params wait_for_status
+minClusterHealthStatus: green
```

Comment:
There are so many custom paramters here that people might want to change other than `wait_for_status`. How about making this more generic and having it be something like:

```
clusterHealthCheckParams: wait_for_status=green&timeout=1s
```

This is going to make this a lot more maintainable if users want to modify the timeout...or add other settings, or if a feature release of Elasticsearch changes the parameters.

Reply:
I've updated to that format.